greenriver/hmis-warehouse
# Refactor Dockerfiles for development and deployment

## Description

Refactor and simplify the dockerfiles. Removes the possibility of stale application files. 

Clear cache by going to github actions -> Cache and deleting the cache for this build (Linux-buildx-warehouse)

## Manual steps

See testing steps. Do a build and then fix your permissions

## How to test

- `docker compose build web`
- `docker compose run -u 0 shell chown -R app-user:app-user /bundle /app /node_modules`
-  Run a web server and shell with the newly built image. Make sure `bundle`, `rails c`, `rake`, etc. all still work
-  Deploy to at least one environment 

## Type of change


## Checklist before requesting review
